com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java

```diff
@@ -169,80 +170,91 @@ private void installOrUpdate(File outputFile) throws Exception {
 			isFragment = fragmentHost != null;
 
 			bsn = bundle.getBsn();
+
+			if(isFragment) {
+				hostBSN =
+						new Parameters(fragmentHost).keySet().iterator().next();
```

Comment:
Do we perform this in the IDE?  where we rrefresh the host on deploy of a fragment?  If so can we just check the code there?  Perhaps it is incorrect as well.